src/coaches/coaches.service.ts

```
@@ -0,0 +1,44 @@
+    return this.db.coach.findMany({
+      orderBy: {createdAt: 'desc'}
+    })
+  }
+
+  async findOneByUsername(username: string): Promise<Coach | null> {
+    const coach = this.db.coach.findUnique({
+      where: { username: username },
+    });
+
+    return coach !== null
+    ? coach
```

ESLint

[error] Insert `··`

(prettier/prettier)